src/Page.test.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import Page from './Page';
+
+describe('Page', () => {
+  const onClickAddTask = jest.fn();
+  const onClickDeleteTask = jest.fn();
+
+  function renderPage({ taskTitle, tasks }) {
+    return render((<Page
+      taskTitle={taskTitle}
+      tasks={tasks}
+      onClickAddTask={onClickAddTask}
+      onClickDeleteTask={onClickDeleteTask}
+    />));
+  }
+
+  const { mainContainer } = render((
```

Comment:
`mainContainer`는 무엇인가요? `container`를 사용하지 않는 이유가 있나요?

```suggestion
  const { container } = render((
```